subsys/partition_manager/Kconfig

```diff
@@ -177,4 +177,11 @@ config PM_SRAM_SIZE
 	default $(dt_node_reg_size_hex,/soc/memory@21000000) if SOC_NRF5340_CPUNET
 	default $(dt_node_reg_size_hex,/soc/memory@20000000)
 
+
+config PM_RESERVE_HW_UNIQUE_KEY
+	bool "Reserve the last flash block for the Hardware Unique Key"
```

Comment:
Based on the comment here: https://github.com/nrfconnect/sdk-nrf/pull/3829/files#r570831209
>  The CONFIG_HUK_LOAD is enabled in b0 and it will fail booting if it cannot load the HUK.

And the fact that you are using a select here: https://github.com/nrfconnect/sdk-nrf/blob/21352ec7b9c72eb00d13bdac75571c9b588fb68b/lib/hw_unique_key/Kconfig#L6

Then why is this symbol configurable by the user ?
In which use-case is the user expected to have `PM_RESERVE_HW_UNIQUE_KEY=y` when having `HW_UNIQUE_KEY_LOAD=n` ?